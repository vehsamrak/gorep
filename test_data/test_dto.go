package test_data

type TestDTO struct {
	Id         int64  `db:"id"`
	Value      string `db:"value"`
	unexported string
}
