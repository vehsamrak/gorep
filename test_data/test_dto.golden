// Code was generated by GoRep. Please do not modify it!

package package_name

type TableTestDTO struct {
    Value string `db:"value"`
    Id int64 `db:"id"`
}
